hacspec/tests/test_machine_integers.rs

```diff
@@ -36,38 +50,133 @@ fn test_signed_public() {
     test_signed_public_macro!(i128);
 }
 
-macro_rules! test_unsigned_secret_macro {
-    ($t:ty) => {
+macro_rules! test_secret_macro {
+    ($t:ty,$true_val:expr) => {
```

Comment:
So what does the `true_val` correspond to here ?

Reply:
That's the bit-mask for true (what I use in `test_integers` as well, this is the predecessor really ...).
For unsigned integers the true value of a bit-mask is the max value, but for signed integers it is -1 (all bits are set 1 in both cases).
This will need documentation....

Reply:
ok, it makes sense :)